src/wms/src/test/java/org/geoserver/wms/wms_1_1_1/GetMapIntegrationTest.java

```diff
@@ -852,6 +852,12 @@ public void testXmlPost() throws Exception {
         checkImage(response);
     }
 
+    @Test
+    public void testXmlPostWithWorkSpaceQualifier() throws Exception {
+        MockHttpServletResponse response = postAsServletResponse("sf/wms?", STATES_GETMAP);
```

Comment:
This is odd... it should not be working, "SF" is the spearfish workspace and topp:states is not in it. I guess the patch is incomplete, maybe it's not setting the current workspace in the catalog?

Reply:
Never mind, in the test states is placed in the "sf`` workspace.